test/e2e/syncer/tunnels_test.go

```diff
@@ -282,6 +245,10 @@ func TestSyncerTunnel(t *testing.T) {
 	labels["state.workload.kcp.io/"+workloadv1alpha1.ToSyncTargetKey(synctargetWsName, syncTarget.Name)] = "Upsync"
 	pod.SetLabels(labels)
 
+	// Try to create the pod in KCP, it should fail because the user doesn't have the right permissions
+	_, err = userKcpClient.Cluster(userWsPath).CoreV1().Pods(upstreamNamespaceName).Create(ctx, &pod, metav1.CreateOptions{})
+	require.Error(t, err)
```

Comment:
Using `EqualError` wold make the test more precise.

Reply:
done! pushed.